tests/test_addon_detail.py

```diff
@@ -1,3 +1,4 @@
+import time
```

Comment:
I've ended up using a `time.sleep()` in the tests below because the screenshot component proved to be a bit challenging:
- most of the elements that are part of this component are already present when the add-on detail page is  loaded, but they're just visually hidden; this means that I didn't have many options for writing explicit waits
- there is one component that changes its state when the screenshot viewer is opened - `_screenshot_viewer_locator = (By.CSS_SELECTOR, '.pswp--open')`, so I tried to use this as my wait element; it didn't work, however, probably because the element is still loaded too fast;  in the end, I've run out of ideas, so I ended up using a time.sleep()